ql/src/java/org/apache/hadoop/hive/ql/plan/VectorPTFDesc.java

```diff
@@ -172,8 +179,8 @@ public VectorPTFDesc() {
 
   // We provide this public method to help EXPLAIN VECTORIZATION show the evaluator classes.
   public static VectorPTFEvaluatorBase getEvaluator(SupportedFunctionType functionType,
-      boolean isDistinct, WindowFrameDef windowFrameDef, Type columnVectorType,
-      VectorExpression inputVectorExpression, int outputColumnNum) {
+      List<ExprNodeDesc> evaluatorParameters, boolean isDistinct, WindowFrameDef windowFrameDef,
```

Comment:
I think we dont need the evaluatorParameters argument passed to this function

Reply:
right, I'm removing this, we can put back later if needed